src/utils/AxiosInstance.ts

```diff
@@ -17,22 +19,48 @@ function getAccountId (): string {
   return accountId
 }
 
+const authApiGwUrl = sessionStorage.getItem('AUTH_API_GW_URL')
+const registriesSearchApiUrl = sessionStorage.getItem('REGISTRIES_SEARCH_API_URL')
+const payApiGwUrl = sessionStorage.getItem('PAY_API_GW_URL')
```

Comment:
Get there here,once, instead of in the interceptor that is called for every network call.